src/DashboardService.Models.Dto/Requests/Board/CreateBoardRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+
+public record class CreateBoardRequest
+{
+  public Guid ProjectId { get; set; }
```

Comment:
it should be either nullable or required

Reply:
+